verity/src/docker/Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM ubuntu:18.04
 
-ARG LIBVDRTOOLS_VERSION
-ARG EVERNYM_CERTIFICATE
+ARG VERITY_VERSION
+ENV VERITY_VERSION=${VERITY_VERSION:-2.19.0.0}
+
+ARG VDRTOOLS_VERSION
+ENV VDRTOOLS_VERSION=${VDRTOOLS_VERSION:-0.8.6}
```

Comment:
please move all the args definitions right before where it is used (before line 26)